src/contexts/states/logFileStore.ts

```diff
@@ -0,0 +1,145 @@
+import * as Comlink from "comlink";
+import {create} from "zustand";
+
+import {Nullable} from "../../typings/common";
+import {CONFIG_KEY} from "../../typings/config";
+import {LOG_LEVEL} from "../../typings/logs";
+import {
+    DO_NOT_TIMEOUT_VALUE,
+    LONG_AUTO_DISMISS_TIMEOUT_MILLIS,
+} from "../../typings/notifications";
+import {QueryResults} from "../../typings/query";
+import {UI_STATE} from "../../typings/states";
+import {SEARCH_PARAM_NAMES} from "../../typings/url";
+import {
+    CursorType,
+    FileSrcType,
+} from "../../typings/worker";
+import {getConfig} from "../../utils/config";
+import {updateWindowUrlSearchParams} from "../UrlContextProvider";
+import useContextStore from "./contextStore";
+import useLogExportStore from "./logExportStore";
+import useLogFileManagerStore from "./LogFileManagerStore";
+import useQueryStore from "./queryStore";
+import useUiStore from "./uiStore";
+import useViewStore from "./viewStore";
+
+
+interface LogFileValues {
+    fileName: string;
+    fileSrc: Nullable<FileSrcType>;
+    numEvents: number;
+    onDiskFileSizeInBytes: number;
+}
+
+interface LogFileActions {
+    setFileName: (newFileName: string) => void;
+    setNumEvents: (newNumEvents: number) => void;
+    setOnDiskFileSizeInBytes: (newOnDiskFileSizeInBytes: number) => void;
+
+    loadFile: (fileSrc: FileSrcType, cursor: CursorType) => void;
+}
+
+type LogFileState = LogFileValues & LogFileActions;
+
+const LOG_FILE_STORE_DEFAULT: LogFileValues = {
+    fileName: "Loading...",
```

Comment:
Similar to the `logData` default value, we want to avoid showing a loading message when users are not loading a file, to avoid confusions. Also, I think by default if we don't set `fileName` as an empty string, the "No file is open." prompt won't show in the left panel, right?
```suggestion
    fileName: "",
```